app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor["name"]} should be vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError(f"{visitor["name"]}"
+                                       f" vaccine is outdatet")
+        if not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError(f"{visitor["name"]} should hame a mask")
```

Comment:
The string formatting here is incorrect. Use single quotes around the key `name` inside the f-string: `f"{visitor['name']} should have a mask"`. Also, there's a typo in the word 'have'.